proxy/create_container_interceptor.go

```diff
@@ -105,7 +110,17 @@ func (i *createContainerInterceptor) setWeaveWaitEntrypoint(container *docker.Co
 		}
```

Comment:
The difference between it not being present in the JSON, and it being an empty list in the JSON.

```Go
	var a []string
	var b = []string{}
	fmt.Println(a == nil) // true
	fmt.Println(b == nil) // false
```